samples/99_matrixexperiments/sycl.cpp

```diff
@@ -694,8 +713,15 @@ static void go_dpas_blockread_vnni_tiled(
         //     auto elapsed = wallclock ? sw_time.count() : hw_time(event);
         //     best = std::min(best, elapsed);
         }
-        auto gops = 2.0 * M * N * K / best / 1e9;
-        printf("Best in %f seconds (%f gops)\n", best, gops);
+
+        
+        double average_event_time = 0.f;
+        for (int i = WARMUP_ITERATIONS; i < total_iterations; i++) {
+            average_event_time += event_times[i];
+        }
+        average_event_time /= (testIterations * 1e3);
+        auto gops = 2.0 * M * N * K;
+        printf("Best in %f microseconds (%f gops)\n", average_event_time, gops / (1e3 * average_event_time));
```

Comment:
Minor: this is the average time now, not the best time:

```suggestion
        printf("Average is %f microseconds (%f gops)\n", average_event_time, gops / (1e3 * average_event_time));
```